### PR TITLE
add document format to quickstart

### PR DESCRIPTION
**Pull request by maryamsulemani97**
Adds document formats to "Add documents" step of the quick start

### CODE REVIEW

File: learn/getting_started/quick_start.md [hunk 1]
```diff
@@ -168,6 +168,10 @@ Congratulations! You're ready to move on to the next step!
 
 For this quick start, we will be using a collection of movies as our dataset. To follow along, first click this link to download the file: <a id="downloadMovie" href="/movies.json" download="movies.json">movies.json</a>. Then, move the downloaded file into your working directory.
 
+::: note
+Meilisearch currently only accepts JSON, NDJSON, and CSV dataset formats, you can read more about them in our [dedicated guide](/learn/core_concepts/documents.md#dataset-format).
```
Comment (dichotommy) [inline learn/getting_started/quick_start.md:172]: Nitpicking, but it's not a dedicated guide if it's not specifically/entirely about that subject 😬

Comment (maryamsulemani97) [inline learn/getting_started/quick_start.md:172]:
```suggestion
Meilisearch currently only accepts JSON, NDJSON, and CSV dataset formats, you can read more about them in our [documents guide](/learn/core_concepts/documents.md#dataset-format).
```

File: learn/getting_started/quick_start.md [hunk 2]
```diff
@@ -168,6 +168,10 @@ Congratulations! You're ready to move on to the next step!
 
 For this quick start, we will be using a collection of movies as our dataset. To follow along, first click this link to download the file: <a id="downloadMovie" href="/movies.json" download="movies.json">movies.json</a>. Then, move the downloaded file into your working directory.
 
+::: note
+Meilisearch currently only accepts JSON, NDJSON, and CSV dataset formats, you can read more about them in our [documents guide](/learn/core_concepts/documents.md#dataset-format).
```
Comment (dichotommy) [inline learn/getting_started/quick_start.md:172]:
```suggestion
Meilisearch currently only accepts data in JSON, NDJSON, and CSV formats. You can read more about this in our [documents guide](/learn/core_concepts/documents.md#dataset-format).
```

**Review by dichotommy — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**